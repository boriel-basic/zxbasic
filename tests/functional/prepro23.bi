' Must return
' PRINT 10;

#define macro(x, y, z) x(y, z)
#define func1(x, y) x(y)
#define func2(x) PRINT x;

macro(func1, func2, 10)



