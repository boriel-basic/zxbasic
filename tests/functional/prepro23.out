#line 1 "prepro23.bi"







PRINT 10;



